db/seeders/items_seeder.rb

```diff
@@ -1,16 +1,12 @@
 # Creates the Base Items
 
 class ItemsSeeder
-  BASE_ITEMS = File.read(Rails.root.join("db", "base_items.json"))
+  BASE_ITEMS = JSON.parse(File.read(Rails.root.join("db", "base_items.json")))
 
   def self.seed
-    items_by_category = JSON.parse(BASE_ITEMS)
-
-    items_by_category.each do |category, entries|
+    BASE_ITEMS.each do |category, entries|
```

Comment:
I think you need `map` here instead of `each`, no? Or maybe not? Is it `BASE_ITEMS` that needs to be returned?